packages/vscode-extension/src/project/metro.ts

```diff
@@ -126,11 +130,12 @@ export class Metro implements Disposable {
 
   public async startInternal(
     resetCache: boolean,
-    progressListener: (newStageProgress: number) => void
+    progressListener: (newStageProgress: number) => void,
+    dependencies: Promise<any>[]
```

Comment:
`any` + why we're adding additional parameter for that here? `dependencies` is typically used for react hooks + we could just await up in the calling code, we don't use and need this argument at all.

Reply:
this is constructed like that to avoid awaiting inside Project.start() to quickly move on to selecting device.